rkt/rkt.go

```diff
@@ -87,7 +87,19 @@ func main() {
 	// Make help just show the usage
 	cmdRkt.SetHelpTemplate(`{{.UsageString}}`)
 
+	// Get the data directory from config
```

Comment:
There are N problems with this code:

1. It doesn't work for custom configuration directories - `getConfig()` tries to get configuration from `globalFlags.SystemConfigDir` and `globalFlags.LocalConfigDir`, but these are not yet set to custom values at this point. That is - `--system-config` and `--local-config` flag haven't been yet parsed.

2. I think that the order of setting `globalFlags.Dir` value should be as follows:
  - If `--dir` parameter is passed, then use this value.
  - If above fails, then try to get the value from configuration.
  - If above fails, then use the default.

  Right now the order is config->parameter->default.

Fixing this might be a small PITA. My proposal is to add a function `getDataDir`, which would implement point 2. To check if --dir was specified, you can use following code:
```go
if f:= rktCmd.PersistentFlags().Lookup("dir"); f != nil {
	if !f.Changed {
		globalFlags.Dir = stuffFromConfig()
	} 
} else {
	panic("Oh my God, you killed --dir")
}
```
And then replace all direct use of globalFlags.Dir with this function.